lib/utilities/monitoringHandler.js

```diff
@@ -190,7 +190,9 @@ function monitoringHandler(clientIP, req, res, log) {
         return monitoringEndHandler(errors.MethodNotAllowed, []);
     }
     const monitoring = (req.url === '/metrics');
-    if (!monitoring) {
+    const healthcheck = (req.url === '/healthcheck');
```

Comment:
Minor: this is a switch with fallthrough on every case, and `MethodNotAllowed` as default.